k8s/cassandra/schema.yaml

```diff
@@ -1,4 +1,41 @@
-applicationApiVersion: v1beta1
+x-google-marketplace:
+  schemaVersion: v2
+
+  applicationApiVersion: v1beta1
+
+  publishedVersion: "$TAG"
+  publishedVersionMetadata:
+    releaseNote: >-
+      A regular update.
+    releaseTypes:
+      - Feature
+    recommended: false
+
+  managedUpdates:
+    kalmSupported: false
```

Comment:
In application.yaml we have (Managed Updates) suffix, but in schema.yaml we have kalm disabled.

Reply:
This diff was due to wrong base. Outdated